README.markdown

```diff
@@ -46,6 +46,12 @@ CFLAGS="-g -O0" ./configure --enable-debug --enable-uzfs=yes
 make
 ```
 
+# Running uzfs
+
+This assumes that you have configured zfs with `--enable-uzfs=yes` option.
+To try zpool and zfs commands, start `cmd/tgt/tgt` binary and leave it running.
```

Comment:
tgt command should be executed with root privileges only else it will give permission error. We should mention this.

Reply:
tgt is just a test program which starts the ioctl server for acting like a kernel to the zfs/zpool commands. As it does not expose any block device, we mapped few signals to the read/write function so that read/write on the created volume can be tested after firing *zfs create* command.

Integration has to be done with *replica* which just need to start the ioctl server (no need to map signals, read the comment above main function in tgt.c). Once it is done, this binary has to be removed. We should use replica for zfs/zpool commands.

I will leave the decision up to you guys, if want to document *tgt* binary here.

Reply:
Thanks Pawan. I agree, in the end it should be replica command which should be described here. Although we are close to have it integrated, we are not there yet. So I will fix sudo problem pointed by Mayank (thanks) and when replica command is integrated, we will change the README file.